LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can we ignore files entirely? For example, I don't think Django Commons would be specifying the license for every project.

Reply:
That's a good idea. The updater can already handle two kinds of licences. Why not disable it completely if required. I could integrate this easily.